doc/plugin.txt

```diff
@@ -4,20 +4,23 @@ Synopsis
 
 Manage project plugins
 
-    add <plugin-spec> [...] ............ Add specified plugins
+    add <plugin-spec> [...] ............ Add specified plugins. If adding a plugin from npm, the
```

Comment:
Too many ifs. 

I was hoping we would actually have an order of resolution of how plugin add picks the plugin in the README.md which is the CLI reference doc. I'm not sure if inline CLI help is the best place for something like this.